reference/cric.v2.yaml

```diff
@@ -1047,6 +1047,172 @@ paths:
+                  type: string
+                slang:
+                  type: string
+                owner:
+                  type: integer
+        description: ''
+  '/collections/{slang}':
+    parameters:
+      - schema:
+          type: string
+        name: slang
+        in: path
+        required: true
+    get:
+      summary: Single Collection
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CollectionModel'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+      operationId: get-collections-id_collection
+      description: Get a single collection.
+    put:
+      summary: Update collection
+      operationId: put-collections-id_collection
+      responses:
+        '200':
+          description: OK
+        '400':
+          description: Bad Request
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+      description: Update a single collection.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                name:
+                  type: string
+                description:
+                  type: string
+                slang:
+                  type: string
+      security:
+        - API Key Authorization: []
+    delete:
+      summary: Delete Collection
+      operationId: delete-collections-id_collection
+      responses:
+        '204':
+          description: No Content
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+      description: Deletes a single collection.
+      security:
+        - API Key Authorization: []
+  '/collection/{slang}/public':
+    parameters:
+      - schema:
+          type: string
+        name: slang
+        in: path
+        required: true
+    post:
+      summary: List of collections
+      operationId: post-collection-id_collection-public
+      responses:
+        '200':
+          description: OK
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+      description: Make the collection public.
+      security:
+        - API Key Authorization: []
+    delete:
+      summary: Make collection private
+      operationId: delete-collection-id_collection-public
+      responses:
+        '200':
+          description: OK
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+      description: Make the collection private
```

Comment:
Description is wrong here. This is for the DELETE.

Reply:
But this one is delete, right?

Reply:
You are right!